### ISSUE TITLE
Default SpringBootTest to "spring.main.banner-mode=off"

### ISSUE THREAD

**Issue by codefromthecrypt** [spring-projects/spring-boot#39583]
Right now, tests driven by `@SpringBootTest` whirr up an application and a side effect is the spring boot logo in the logs. This isn't so much a problem the first time, but if you write a lot of tests, you'll quickly drown your console or CI logs with the logo. Let's default it to "spring.main.banner-mode=off", or if it is quite important to show the logo, some how rate limit it to once per run!

**Comment by mnhock** [comment]
I deactivate the banner for my test profile in the configuration.

**Comment by codefromthecrypt** [comment]
> I deactivate the banner for my test profile in the configuration.

yep while every project can do this, the request is about if this can be made default, so we don't have to. Currently, I also do this, but sometimes I work on other projects who don't and see tons of banners fly by which hide higher value lines.

**Comment by quaff** [comment]
`@SpringBootTest` is not enough, test slices like `@JdbcTest` should adapt also.

**Comment by philwebb** [comment]
We discussed this today as a team and we're not keen to change the default banner display. We feel it's better for `@SpringBootTest` to work as closely as possible to the real application. We've also found that the banner helps folks to understand application context caching in tests.

We do think that it would be nice if folks could disable the banner easily if they want to, but we think that #38098 will provide a good way to do that when it's implemented.

In the meantime, you might be able to add an empty `banner.txt` file to `src/test/resources` or consider tuning your build system so that console output is redirected to file. Gradle does this by default and it gives you clean console output but still access to the logs when reviewing the test report.
